apps/dashboard/src/components/container/ContainerActionsDialog.vue

```diff
@@ -104,73 +65,35 @@ const header = computed(() => {
   return '';
 });
 
-const creationSchema = toTypedSchema(
-    yup.object({
-      name: yup.string().required(),
-      public: yup.boolean().required().default(true),
-      owner: yup.mixed<BaseUserResponse>().required()
-    })
-);
+const form = schemaToForm(containerActionSchema);
 
-const { defineField, handleSubmit, errors, resetForm } = useForm({
-  validationSchema: creationSchema,
-});
-
-const [name, nameAttrs] = defineField('name');
-const [publicContainer, publicAttrs] = defineField('public');
-const [owner, ownerAttrs] = defineField('owner');
-
-const closeDialog = () => {
-  name.value = '';
-  publicContainer.value = undefined;
-  owner.value = undefined;
-  resetForm();
-  emit('update:visible', false);
+const openDialog = () => {
+  if (props.container) updateFieldValues(props.container);
 };
 
-const handleSubmitContainer = handleSubmit(async (values) => {
-  // Updating a container
-  if (props.container) {
-    await containerStore.updateContainer(
-        props.container.id,
-        { name: values.name,
-        public: values.public,
-        products: props.container.products.map((p) => p.id) }
-    ).then(() => {
-      closeDialog();
-      toast.add({
-        severity: 'success',
-        summary: t('successMessages.success'),
-        detail: t('successMessages.updateContainer'),
-        life: 3000,
-      });
-    }).catch((err) => handleError(err, toast));
-  } else {
-    // Creating a container
-    const container = await containerStore.createEmptyContainer(
-        values.name,
-        values.public,
-        values.owner.id
-    ).catch((err) => handleError(err, toast));
-
-    if(!container) return;
-
-
-
-    // Adding container directly to associated POS
-    if(props.associatedPos) {
-      const posStore = usePointOfSaleStore();
-      await posStore.updatePointOfSale(props.associatedPos.id, {
-        name: props.associatedPos.name,
-        useAuthentication: props.associatedPos.useAuthentication,
-        containers: [container, ...props.associatedPos.containers].map(c => c.id),
-        id: props.associatedPos.id,
-        cashierRoleIds: props.associatedPos.cashierRoles.map(c => c.id)
-      });
+const closeDialog = () => {
+  visible.value = false;
+  form.context.resetForm({
+    values: {
+      name: '',
+      owner: undefined,
+      public: false
     }
+  });
+  console.log(1);
+};
 
-    emit('create:container', container);
-    closeDialog();
+setSubmit(form, form.context.handleSubmit(async (values) => {
+  if(state.value.create) {
+    const createContainerRequest: CreateContainerRequest = {
+      name: values.name,
+      ownerId: values.owner.id,
+      products: [],
+      public: values.public
+    };
+
+    await containerStore.createContainer(createContainerRequest)
+        .catch((err) => handleError(err, toast));
     toast.add({
       severity: 'success',
       summary: t('successMessages.success'),
```

Comment:
Where is the actual success message?

Reply:
yeah below it, but I placed in the then now, because that makes more sense :)